user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -132,7 +132,10 @@ def get_supported_gpus(self) -> dict:
                 memory_mb = num_cpu * unit_info['memPerCPU']
                 sys_info_obj = SysInfo(num_cpus=num_cpu, cpu_mem=memory_mb)
                 # create gpu_info
-                gpu_cnt = 2 if num_cpu >= 16 else 1
+                # TODO Should we use only 1 GPU for machines with cores LT 16
+                # TODO move the constant to the static dataproc configuration file
+                # pu_cnt = 2 if num_cpu >= 16 else 1
+                gpu_cnt = 2
```

Comment:
Why did this logic change to default to 2 for everywhere?  Logic before had 1 GPU if cores were less than 16.